modules/ROOT/pages/deploy-to-runtime-fabric.adoc

```diff
@@ -50,25 +55,34 @@ This tab displays the name and type of the deployment target you selected above.
 
-. To enforce the deployment of replicas across different nodes, select *Enforce deploying replicas across nodes*. This option is only available for Runtime Fabric version 1.5.25 or later.
+. To enforce the deployment of replicas across different nodes, select *Enforce deploying replicas across nodes*.
 +
 [WARNING]
+====
 This can cause some replicas in the deployment to remain in *Pending* status if there are unsufficient worker nodes or resources available on the worker nodes needed to deploy each replica. The deployment status will transition to *Running* after all replicas are deployed on different worker nodes.
+====
 
 . Specify the number of *Cores* and amount of *Memory* to allocate to each replica of the application by adjusting the sliders to the appropriate values.
 +
 See how to xref:deploy-resource-allocation.adoc[determine resource allocation] for applications deployed to Runtime Fabric.
 
 . Specify network configuration for the application:
-
++
 .. Click the *Ingress* tab.
 .. Enable or disable the following options:
 +
-* *Enable inbound traffic to this application* to allow incoming HTTP requests to be processed by this application. Ensure your application is configured to accept connections on port 8081.
+* *Enable inbound traffic to this application* to allow incoming HTTP requests to be processed by this application. Ensure your application is configured to accept connections on port 8081. 
++
+If you are using Runtime Fabric on Self-managed Kubernetes, this option is enabled by default. To send requests to a deployment using a different supported environment, add additional endpoints for this Runtime Fabric. See xref:enable-inbound-traffic-self.adoc[Configure Domains and Inbound Traffic (RTF Self-managed Kuberenetes)]
++
+If you are using Runtime Fabric on VMs / bare metal, you must enable inbound traffic manually. Additional endpoints are sourced from the subject alternative names (SAN) certificate property. If a certificate has multiple DNS entries specified by the SAN property, the available URLs are displayed in the *Application URL
 +
 [NOTE]
 To access a Runtime Fabric instance using more than one DNS, add additional DNS entries in the subject alternative names (SAN) certificate property. If a certificate has multiple DNS entries specified in the SAN property, the available URLs are displayed in the *Application url* section.
-* *Enable Last-Mile security* to forward HTTPS connections to be decrypted by this application. This requires an SSL certificate to be included in the Mule application, and also requires more CPU resources.
++
+* *Enable Last-Mile security* to forward HTTPS connections to be decrypted by this application. This requires an SSL certificate to be included in the Mule application, and also requires more CPU resources. This option is only applicable to Runtime Fabric on VMs / baremetal.
++
+* *Forward SSL session* This option is only applicable to Runtime Fabric on VMs / baremetal.
```

Comment:
This option is only available on VM / bare metal, but is missing from the current docs.

Reply:
Info provided in slack channel.